_services_projects/forest-service.md

```diff
@@ -20,50 +20,25 @@ product_clients:
 
 <div class="small-caps">Approach</div>
 ### Breaking the work into modules
 
-Before helping Forest Service hire private vendors to build a permit
-tool, 18F conducted a series of workshops with the Forest Service to
-investigate the business process behind permit applications. With that
-information, we built a small prototype of the type of online permit
-tool that could address their issues. We tested the prototype with both
-the public and Forest Service employee users and used that research to
-draft an RFP using 18F
+Before helping Forest Service hire vendors to build a permit tool, we conducted a series of workshops with the Forest Service to investigate the business process behind permit applications. With that information, we built a small prototype of the type of online permit tool that could address their issues. We tested this prototype with both public and Forest Service staff users and used that research to draft an RFP using 18F
 [Acquisition’s Agile Blanket Purchase Agreement](https://18f.gsa.gov/what-we-deliver/agile-bpa/) (BPA).
 
-Instead of putting the entire permit application tool into one RFP, the
-team started by asking only for an API to connect to a legacy system
-within the Forest Service. By breaking the work into pieces (called
-modules) the Forest Service was able to get functionality quickly while
-reducing risk on each module.
+Instead of putting the entire permit application tool into one RFP, we started by building an API to connect to a legacy system within the Forest Service. By breaking the work into pieces (called modules) the Forest Service was able to get functionality quickly while reducing risk on each module.
 
-Working with the Forest Service and the vendor, we built the first API
-module in four months. We provided technical and design guidance to the
-vendor, and strategies for the Forest Service to serve as empowered
-product owners.
+Working with the Forest Service and the vendor, we built the initial API module in four months. We provided technical and design guidance to the vendor as well as coaching and strategies for the Forest Service to serve as product owners.
 
-With the API module in hand and working well, we then helped the Forest
-Service draft, award, and manage two additional RFPs to build a
-public-facing application and a module to support Christmas Tree
-permits. We also integrated the application with
-[login.gov](http://login.gov) so users could securely log in to the
-system.
+With the API module in hand and working well, we then helped the Forest Service draft, award, and manage additional RFPs to build a public-facing application, a module to support Christmas tree permits, and a module to support Special Uses permits. We also integrated the application with [login.gov](http://login.gov) so users could securely log in to the system and [Pay.gov](https://pay.gov) for credit card payment processing.
```

Comment:
Suggestion: "With the API module in hand and working well, we then helped the Forest Service draft, award, and manage additional RFPs to build a public-facing application including a module to support Christmas tree permits and a module to support Special Uses permits. "